build.psm1

```diff
@@ -439,10 +439,9 @@ Fix steps:
     # Add --self-contained due to "warning NETSDK1179: One of '--self-contained' or '--no-self-contained' options are required when '--runtime' is used."
     if ($Options.Runtime -like 'fxdependent*') {
         $Arguments += "--no-self-contained"
-        # The UseAppHost = false property avoids creating ".exe" for the fxdependent packages.
-        # The ".exe" is not a cross-platform executable, but specific to the platform that it was built on.
-        # We do not need to ship that.
-        $Arguments += "/property:UseAppHost=false"
+        # The UseAppHost = true property creates ".exe" for the fxdependent packages.
+        # We need this in the package as Start-Job needs it.
+        $Arguments += "/property:UseAppHost=true"
```

Comment:
I think we removed this because it was not signed.. we should sign it.

Reply:
Good point.. let me check the test build.

Reply:
Updated the packaging code.